config/packages/monolog.yaml

```diff
@@ -54,8 +54,4 @@ when@prod:
             console:
                 type: console
                 process_psr_3_messages: false
-                channels: ["!event", "!doctrine"]
-            deprecation:
-                type: stream
-                channels: [deprecation]
-                path: php://stderr
```

Comment:
I think we want to keep this so containers display error output in AWS.

Reply:
that removal was an accident. un-removed.

but with that config restored, it turns out that the other change doesn't do what i thought it would. i'm back to reading the manual...